### DISCUSSION TITLE
How do I order foreign table results?

### DISCUSSION THREAD

**Discussion by voicechains** [supabase/supabase#14964] [Questions] [answered]
Is there any way to specify a column name to sort the `foreign` table by? My comments are returning out of order. It seems to be sorting the `foreign` Table (comments) by `id` .

**Posts (parent)**
`id` (uuid)
`created_at`
`title`

**Comments (child)**
`id`  (uuid)
`created_at`
`post_id`
`message`

```
supabase
  .from('posts')
  .select('title, comments(created_at, message)')
  .eq('post_id', id)
  .order('created_at', { foreignTable: 'comments(created_at)' })
```

The above code isn't working. What am I doing wrong?

**Comment by GaryAustin1** [comment, marked as answer]
The syntax is just `{foreignTable: 'comments'}`  You specified the column with 'created_at' at the start.
Also if created_at is the same for comments then their order will be random as Postgres has no natural order.

**Reply by voicechains** [reply]
Seems to be working now. thank you!

**Comment by voicechains** [comment]
Resolved
